src/scicat_online_ingestor.py

```diff
@@ -88,13 +88,16 @@ def build_online_config() -> OnlineIngestorConfig:
         Path(arg_namespace.config_file), arg_namespace
     )
 
-    return build_dataclass(OnlineIngestorConfig, merged_configuration)
+    return build_dataclass(
+        tp=OnlineIngestorConfig, data=merged_configuration, logger=logger, strict=False
+    )
 
 
 def main() -> None:
     """Main entry point of the app."""
-    config = build_online_config()
-    logger = build_logger(config)
+    tmp_config = build_online_config()
+    logger = build_logger(tmp_config)
+    config = build_online_config(logger=logger)
```

Comment:
I had to build the configuration twice in order to include the error messages in the logging
but I think it consumes trivial amount of time to build it twice so it should be fine...